manager/manager.go

```diff
@@ -779,21 +777,31 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 			return err
 		}
 		unencryptedKeyBlock := &pem.Block{
-			Type:    keyBlock.Type,
-			Bytes:   unencryptedKey,
-			Headers: keyBlock.Headers,
+			Type:  keyBlock.Type,
+			Bytes: unencryptedKey,
 		}
 
-		// We were able to decrypt the key, but with the previous passphrase. Let's encrypt
-		// with the new one and store it in raft
-		finalKey, err = ca.EncryptECPrivateKey(pem.EncodeToMemory(unencryptedKeyBlock), strPassphrase)
+		// we were able to decrypt the key with the previous passphrase - if the current passphrase is empty,
+		// the we store the decrypted key in raft
+		finalKey = pem.EncodeToMemory(unencryptedKeyBlock)
+
+		// the current passphrase is not empty, so let's encrypt with the new one and store it in raft
+		if strPassphrase != "" {
+			finalKey, err = ca.EncryptECPrivateKey(finalKey, strPassphrase)
+			if err != nil {
+				log.G(ctx).Debugf("failed to rotate the key-encrypting-key for the root key material of cluster %s", clusterID)
```

Comment:
Would it make sense to include the error returned by `EncryptECPrivateKey` here?